docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,129 @@
+
+### Sizing options
+
+Size props are optional in `GatsbyImage` and `StaticImage`. Because the images are processed at build time, the plugin knows the size of the source image and can add the correct width and height to the `<img>` tag, so it displays correctly with no layout jumping. However, if you want to change the display size you can use the size options to do this.
+
+#### `width`/`height`
+
+_Layouts: fixed and constrained_
+
+For a `fixed` layout, these define the size of the image displayed on screen. For a `constrained` image these define the maximum size, as the image will scale down to fit smaller containers if needed.
+
+If you set just one of these, the source image is resized to that width or height while maintaining aspect ratio. If you include both then it is also cropped if needed to ensure it is that exact size.
+
+#### `aspectRatio`
+
+_Layouts: fixed, constrained, and fullWidth_
+
+The `aspectRatio` prop forces an image to the specified aspect ratio, cropping if needed.
+
+For `fixed` and `constrained` images, you can also optionally pass either a `width` or `height`, and it will use that to calculate the other dimension. For example, if you pass `width={800} aspectRatio={4/3}` then `height` will be set to the width divided by the aspect ratio: so `600`. Passing `1` as the aspectRatio will crop the image to a square. If you don't pass a width or height then it will use the source image's width.
+
+For `fullWidth` images you don't specify width or height as it resizes to fit the screen width. Passing `aspectRatio` will crop the image if needed, and the height will scale according to the width of the screen. For example, if you set the aspectRatio to `16/9` then when the image is displayed full width on a screen that is 1024px wide, the image will be 576 pixels high.
+
+> There are several advanced options that you can pass to control the cropping and resizing behavior. For more details, see the advanced options reference.
+
+### Placeholder
+
+**Prop name: `placeholder`**
+
+Gatsby image components are lazy-loaded by default, which means that if they are offscreen they are not loaded by the browser until the come into view. To ensure that the layout does not jump around, a placeholder is displayed before the image loads. There are three types of placeholder that you can choose:
+
+#### Dominant color
+
+_Component prop: `"dominantColor"`. Resolver prop: `DOMINANT_COLOR`_
```

Comment:
If we end up using a table for the `layout` values, we should use a similar table here for the `placeholder` values.